starknet_py/tests/unit/hash/transaction_test.py

```diff
@@ -175,7 +149,7 @@ def test_compute_declare_v3_transaction_hash(common_data, declare_data, expected
                 ],
                 "account_deployment_data": [],
             },
-            0x15F2CF38832542602E2D1C8BF0634893E6B43ACB6879E8A8F892F5A9B03C907,
+            0x119386B4AAAEF905BF027D3DD2734474C5E944942BF3FBD8FDB442704D32B8B,
```

Comment:
note: Resource bounds affect the calculation of the transaction hash, hence its change.